src/main/java/frc/robot/Constants.java

```diff
@@ -253,7 +253,7 @@ public static final class SpeakerConstants {
     public static final double kMaxGoodShootingDistance = 3.56;
     public static final double kMaxPossibleShootingDistance = 4.01;
 
-    public static final double kAngularTolerance = Units.degreesToRadians(1.0);
+    public static final double kAngularTolerance = Units.degreesToRadians(5.0);
```

Comment:
I'm not sure we want to go as high as 5 here. Let's try something more like 2 or 3 first.